Cargo.toml

```diff
@@ -181,7 +181,7 @@ cairo-lang-test-runner = { git = "https://github.com/starkware-libs/cairo", rev
 cairo-lang-test-utils = { git = "https://github.com/starkware-libs/cairo", rev = "76e8c943895c904da121fb0f6e48551d367e64d8" }
 cairo-lang-utils = { git = "https://github.com/starkware-libs/cairo", rev = "76e8c943895c904da121fb0f6e48551d367e64d8" }
 cairo-language-server = { git = "https://github.com/software-mansion/cairols", rev = "7b26a0edc124028781a92e68de24882faea0d885" }
-cairo-lint-core = { git = "https://github.com/software-mansion/cairo-lint", rev = "136ae392703c6ee6a8ad41b7b5e1329b8b9a7604" }
+cairo-lint-core = { git = "https://github.com/software-mansion/cairo-lint", rev = "dfbc8e5fcc2b75a85781da04559d2a093b96eb3b" }
```

Comment:
FYI: will be changed as soon as we merge https://github.com/software-mansion/cairo-lint/pull/235